src/views/InteractiveMap/InteractiveMap.vue

```diff
@@ -90,6 +90,7 @@
           :card="card"
           :isSelected="card === selectedCard"
           :isOnlyCard="cards.length === 1"
+          :withDialog="card.withDialog"
```

Comment:
No need to pass this - it's already available to the component through `this.card.withDialog`